src/ui/components/shared/IconWithTooltip.module.css

```diff
@@ -0,0 +1,4 @@
+.iconWithTooltipButton div {
```

Comment:
I don't think this selector matches up with your markup where it's applied to the `button` and is wrapped by a div. As it stands, I don't think this selector is doing anything because it doesn't match anything

Reply:
Thanks! I read the code the same way, but when I add a border to that selector, I see it show up. So it does seem to be finding it.

Reply:
I agree that this styling is confusing. The `{icon}` in this case is rendered within a `<div>` which is why this style works, but it should probably be set on the div instead.